client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	require( 'test/helpers/use-fake-dom' )();
```

Comment:
I've verified, and it still works as expected... so it's ok for me to change.
Could you also please explain what you mean with 

>Unless we're running into load order issues

Did you have problems in the past, which were caused by requiring `use-fake-dom` at the top level?